src/BenchmarkDotNet.Core/BenchmarkDotNet.Core.csproj

```diff
@@ -23,6 +34,17 @@
     <PackageReference Include="System.Xml.XPath.XmlDocument" Version="4.3.0" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'uap10.0' ">
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="5.3.2" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.3.0" />
+    <PackageReference Include="System.Xml.XPath.XmlDocument" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
+    <PackageReference Include="System.Threading" Value="4.3.0" />
+    <PackageReference Include="System.Reflection" Value="4.3.0" />
+    <PackageReference Include="System.Console" Value="4.3.0" />
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Value="4.3.0" />
```

Comment:
Value is not a valid property in PackageReference. You have to use Version instead of Value for these 4 packages.